open-apparel-registry/open-apparel-registry
# Add footer in embed mode

## Overview

Display the footer, which has a different design and contents to the main footer, when the map is in embed mode.

Connects #1336 

## Demo

![image](https://user-images.githubusercontent.com/1042475/118498981-897edc00-b6f4-11eb-9a17-30a289ac2278.png)

## Testing Instructions

- View the app in embed mode, and verify the embed footer is displayed. 
- Ensure the app 

## Checklist
